_posts/2015-11-09-scrum-basics-2.md

```diff
@@ -0,0 +1,118 @@
+And yes, this includes developers and PO.
+One example that most software developers probably understand is the problem of scope creep.
+The developers are working on their feature, when some manager steps into the room and forces someone to take care of an ultra-urgent topic.
+The developer is distracted from his actual task and since it was a manager who told them to go work on something else, they usually do it.
+Not in Scrum!
+In Scrum it would be the task of the Scrum Master to step inbetween the manager and the developer.
+The Scrum Master would tell the manager politely to go and talk to the PO, who will create a ticket for his request and sort it into the Backlog according to its business value.
+If it really is ultra-urgent, it will be solved within the next sprint.
+Only in real blocker situations (like a live-platform being down) a running sprint would be interrupted.
+And ONLY the PO has the right to decide this.
+
+That little story already leads to the third biggest part of the Scrum Masters time: Coaching.
+He is responsible to coach not only the team and the PO on Scrum and agile development topics, but also all other departments in the company that have to interact with the Scrum Teams.
+They need to know the rules to being able to follow them.
+Normally they also need a constant reminder of those rules.
+Coaching the Product Owner is another big topic.
+As you have already learned, the PO role is quite complex and there are several techniques that can help him in his daily business.
+But someone needs to teach him those techniques and also make him aware of problems.
+Only if someone tells the PO that something is not working, they can improve themselves and their work.
+The same goes for the Development Team.
+It’s the SM's job to constantly push them to improve themselves, to become more self-organised and to be open for changes.
+
+You may have noticed that I switched from the first to the third biggest part of the Scrum Master's job, so here is the second: They have to ensure that impediments are removed.
+Impediments are all things that block the team from performing.
+If it is the manager in the room, a computer that’s too slow or a suddenly unusual high amount of support tickets: The Scrum Master needs to investigate and find a solution.
+That doesn’t mean that they have to solve everything by themselves.
+They only need to find the responsible people and ensure that they are solving the problem.
+Like pushing the decision maker for a fast “go” on a new computer to replace the slow one.
+To order and install it is then up to IT.
+Or the SM would have to investigate why there are suddenly so many support tickets.
+Has the quality dropped?
+Or is there a high sick-leave rate in the Service Center so they aren’t able to work in detail on every ticket and just pass them on?
+The SM would need to talk to them to know, if the Development Team might plan less tickets in the next sprint in order to have more support time.
```

Comment:
The SM would need to talk to them to understand if the Development Team should plan to do less tickets in the next sprint in order to have more support time.